### DISCUSSION TITLE
Setting animation counts

### DISCUSSION THREAD

**Discussion by go4cas** [unocss/unocss#2706] [Q&A]
In the preset definition, animations like `animate-pulse` has `count` set to `infinite`. Is there a class to define the count, e.g. to only pulse 10 times? If not, is there some way to achieve this with the `uno.config.ts`?

**Comment by go4cas** [comment]
Not sure if this is the best approach, but I ended up adding this to the `rules` inside `uno.config.ts`:
```javascript
[/^pulse-(\d+)$/, ([, d]) => ({ 'animation-iteration-count': d })]
```

Then in any component, I can add the dynamic class: `class="animate-pulse pulse-3"`

**Comment by zyyv** [comment]
We already has this rule. https://github.com/unocss/unocss/blob/d45bcb91690fae24b99d3accdf929135858a28df/packages/preset-wind/src/rules/animation.ts#L65
Or you can rewrite that animate theme to override it. [Playground](https://unocss.dev/play/?html=AQHgJglgbsCGB2EC2sAuBTAtABwK4BsBndAPgChhLd4B7AY0MIpAHpIpyg&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmANHGFKgM6owCCMMUwARgK4zDoCeZF1tAKpYIcAL5x0UCCDgByNiIDGVKnIBQ61AA9IsFBgCGbADbw0mHPkLAiACgTq45SjRhUAXHADa-N8Ig7AEo%2BV1oGJlYOLm5ggF0SJzgYAAtUEFQPR2dnQyxQQ04CLKScuCUIRXcSsrKwUxovAEYABlKcsXbO506xIPUgA&css=Q&options=N4IgLgTghgdgzgMwPYQLYgFwKgGzgUwF8g)

**Comment by go4cas** [comment]
Great! Thanks for pointing this out! I removed my custom rules, and can now just use `animate-3`.
